run-tests.sh

```diff
@@ -27,5 +27,5 @@ pydocstyle invenio_migrator tests && \
 isort -rc -c -df **/*.py && \
 check-manifest --ignore ".travis-*" && \
 sphinx-build -qnNW docs docs/_build/html && \
-python setup.py test && \
+py.test tests/unit/ && \
```

Comment:
The test for legacy should only run for a given combination on travis, it is more or less like for the end2end tests.

Maybe we can name it differently 😄